script/setup/install-critools

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```

Comment:
why change bash to sh?

Reply:
Bash has roughly 10 times the footprint (disk/memory/etc.) as compared to standard small POSIX shells like e.g. dash (which is the default /bin/sh in e.g. Debian since some releases). Unless there is some specific bash functionality that is explicitly needed & used, there is no reason to bring on that huge beast for a shell when things could be kept POSIX (and hence /usr/bin/env sh be used instead of explicitly stating bash) and small/efficient.

Reply:
Go SDKs already have huge footprint. So requiring bash doesn't seem problematic.

Reply:
Using unnecessary resources for nothing is simply not good programming practice, for good reasons. It all adds up.